### PR TITLE
Refactored grabValueFrom and seeInFields

### LINKED ISSUE #2378: seeInField doesn't see value of disabled form fields
Using the `seeInField` function on a disabled form field or a form field within a disabled field set always leads to an empty value for that form field. This makes it impossible to test for a given value in a disabled field. This did work in version 2.0.12 but is not working anymore in the latest version 2.1.2.

### PR DESCRIPTION
**Pull request by DavertMik**
This should reduce code duplication and allow using `seeInField` and `dontSeeInField` with disabled form fields. Fixes #2378 

@zbateson @Naktibalda please review

### CODE REVIEW

File: tests/data/app/db [hunk 1]
```diff
@@ -1 +1 @@
-a:0:{}
```
Comment (Naktibalda) [inline tests/data/app/db]: This file should be added to .gitignore, right?

**Comment by Naktibalda** [comment]
I don't see anything wrong in this change.

File: tests/data/app/db (hunk 1 above)
Comment (DavertMik) [inline tests/data/app/db]: probably

File: src/Codeception/Lib/InnerBrowser.php [hunk 2]
```diff
@@ -996,35 +968,45 @@ public function grabValueFrom($field)
...
-                    $val = $option->getAttribute('value');
-                    if (!$isMultiple) {
-                        return $val;
-                    }
-                    $results[] = $val;
-                }
+            $field = new ChoiceFormField($nodes->filter('select')->getNode(0));
+            $options = $nodes->filter('option[selected]');
+            foreach ($options as $option) {
+                $values[] = $option->getAttribute('value');
```
Comment (DavertMik) [inline src/Codeception/Lib/InnerBrowser.php]: the strange thing that I couldn't use `$field->getValue()` here as it returned all options no matter if they were selected or not :(

**Pull request merged**